gptables/test/test_wrappers.py

```diff
@@ -243,6 +243,14 @@ def test__write_empty_table(self, testbook, create_gptable_with_kwargs):
         with pytest.raises(ValueError):
             testbook.ws._write_table_elements([0,0], gptable, auto_width=True)
 
+    def test__write_integer_table(self, testbook, create_gptable_with_kwargs):
+        gptable = create_gptable_with_kwargs({
+            "table": pd.DataFrame({"a": [0], "b": [1]})
+        })
+
+        #Testing that this function executes with no errors
```

Comment:
Thanks for adding a comment to make this more explicit. Another option might be to use [does_not_raise](https://docs.pytest.org/en/6.2.x/example/parametrize.html#parametrizing-conditional-raising) but for this situation that might be overkill